### PR TITLE
.github: add failing_test_jenkins_template form for filing CI bugs

### PR DESCRIPTION
**Pull request by qmonnet**
Use a form to make it easier to file issues for CI flakes on Jenkins, by having a form where we can just copy-paste the different messages (stack, stdout, stderr) without having to care about the formatting.

I have not found a way to place the generated content inside of `<details><summary>...</summary> ... </details>` blocks.

I pushed on my fork of the Cilium repo, and a preview of the form can be found at https://github.com/qmonnet/cilium/issues/new/choose (labels are not set because I haven't created these labels on the fork).

### CODE REVIEW

**Review by nbusseneau — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/failing_test_jenkins_template.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      description: Standard output messages for the failure, as reported by Jenkins. This will be automatically formatted into code, so no need for backticks.
+      placeholder: |
+        Cilium pods: [cilium-9gp4w cilium-vpsn6]
+        Netpols loaded:
+        CiliumNetworkPolicies loaded: default::fqdn-proxy-policy.yaml
+        Endpoint Policy Enforcement:
+        Pod   Ingress   Egress
+        Cilium agent 'cilium-9gp4w': Status: Ok  Health: Ok Nodes "" ContinerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 37 Failed 0
+        Cilium agent 'cilium-vpsn6': Status: Ok  Health: Ok Nodes "" ContinerRuntime:  Kubernetes: Ok KVstore: Ok Controllers: Total 33 Failed 0
+      render: shell
```
Comment (nbusseneau) [inline .github/ISSUE_TEMPLATE/failing_test_jenkins_template.yaml:65]: I wonder if there is a renderer available for displaying in `<details><summary>...</summary> ... </details>` blocks?

Comment (qmonnet) [reply .github/ISSUE_TEMPLATE/failing_test_jenkins_template.yaml:65]: I wondered too, but I haven't been able to find a list of the existing renderers :/. I could look further, or reach out to support.
But even then, we'd need a renderer that does _both_ the `<details>` block and the code formatting.

**Review by pchaigno — APPROVED** [review]
Nice :rocket:!

**Review by aditighag — APPROVED** [review]
Thanks!

**Pull request merged**